addons/website_forum/controllers/main.py

```diff
@@ -86,8 +89,17 @@ def questions(self, forum, tag=None, page=1, filters='all', sorting=None, search
             domain += [('tag_ids', 'in', tag.id)]
         if filters == 'unanswered':
             domain += [('child_ids', '=', False)]
-        elif filters == 'followed':
+        if filters == 'solved':
+            domain += [('has_validated_answer', '=', True)]
+        if filters == 'unsolved':
+            domain += [('has_validated_answer', '=', False)]
+
+        if my == 'mine':
```

Comment:
user_id = request.env.user.id

and you can after replace the 3 occurences here

Reply:
Ok, but I'm replacing with:
`user = request.env.user` rather than `user_id = request.env.user.id`, 

because of
```python
if my == 'followed':
    domain += [('message_partner_ids', '=', request.env.user.partner_id.id)]
```